lib/ui/window.dart

```diff
@@ -284,22 +284,36 @@ class FlutterView {
   external static void _updateSemantics(SemanticsUpdate update);
 }
 
-/// A [FlutterView] that includes access to setting callbacks and retrieving
-/// properties that reside on the [PlatformDispatcher].
+/// Deprecated. Will be removed in a future version of Flutter.
```

Comment:
I'm proposing to change this to simply:
```
/// A deprecated property that used to represent the [FlutterView] that backs the 
/// global `window` singleton, which is also deprecated.
///
/// This deprecation is to prepare for Flutter's upcoming support for
/// multiple views and eventually multiple windows. To migrate, see
/// <link>.
```

The idea is that adding a "simpler" migration guide here does not help much in my opinion, since the developer will have to check the full guide anyway.

Reply:
I prefer not requiring people to context switch and having them go to the migration guide if we can present the most important information right here.